Dalamud/Game/Text/XivChatEntry.cs

```diff
@@ -32,3 +32,34 @@ public sealed class XivChatEntry
     /// </summary>
     public bool Silent { get; set; }
 }
+
+/// <summary>
+/// This class represents a raw chat message.
+/// </summary>
+public struct XivChatEntryRaw()
+{
+    /// <summary>
+    /// Gets the type of entry.
+    /// </summary>
+    public XivChatType? Type { get; set; }
+
+    /// <summary>
+    /// Gets the message timestamp.
+    /// </summary>
+    public int Timestamp { get; set; }
+
+    /// <summary>
+    /// Gets the sender name.
+    /// </summary>
+    public byte[] Name { get; set; } = [];
+
+    /// <summary>
+    /// Gets the message.
+    /// </summary>
+    public byte[] Message { get; set; } = [];
```

Comment:
Are these the only two methods for chat entries, and are they always guaranteed to be SeStrings? If so, you can add these fields to `XivChatEntry` as `NameBytes` and `MessageBytes`, then have `Name`/`Message` read from/write to accordingly.